host/host.js

```diff
@@ -96,7 +126,7 @@ Host.create = function() {
   var pythonChild = spawnVirtualEnv(
     'gaia-integration',
     ['--path=' + socketPath],
-    { stdio: [0, 1, 2, 'pipe'] }
+    { stdio: ['pipe', 'pipe', 'pipe', 'pipe'] }
```

Comment:
Pretty sure you want 0, 1, 2 so they correctly inherit fd did you change this for some reason? (to avoid seeing python output?)